config/ax_code_coverage.m4

```diff
@@ -202,6 +202,10 @@ code-coverage-clean:
 # $(PACKAGE_VERSION). In order to add the current git hash to the title,
 # use the git-version-gen script, available online.
 
+# Add any folders you want to ignore here
+# Ignore tmp and tests themselves
+CODE_COVERAGE_IGNORE_PATTERN ?= "/tmp/*" "*/tests/*"  
```

Comment:
I'm curious why this was moved up?  Was it needed for some reason or purely cosmetic?

Reply:
Mainly cosmetic.

On row 209 it lists states:
`# Optional variables`

Most of which are not actually used by the current code.
It seems prudent to have non-optional variable that are actually used, before (and clearly seperated from), optional-unused variables.

However: If you prefer I could move the whole blob to row 222 and just use some extra spacing to make it stand out.

Reply:
@behlendorf Moved it back down anyway.
In hindsight it looks just as clean... 👍